context.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ga4gh
+
+import "context"
+
+type key int
+
+const identityKey key = 0
+
+// NewContext creates a new context.Conext from ctx that carries identity.
+func NewContext(ctx context.Context, identity *Identity) context.Context {
```

Comment:
I think these are usually called WithIdentity or NewIdentityContext or similar

Reply:
It's true but all of those have useful package names: userip.NewContext or trace.NewContext etc - you have ga4gh.NewContext (but what's a ga4gh context?)